test/performance/perf_test.lua

```diff
@@ -101,15 +101,19 @@ local row_name = {
     insert = 'insert',
```

Comment:
There is no way to run particular test and look at partial report:

```shell
$ PERF_MODE_ON= luatest -v perf.test_crud_pairs_gt_with_stats_disabled
```

Reply:
Fixed. Also it should be noted that `-c` is required since `-v` do not captures printout.

Reply:
That's nice: should be quite useful for debugging performance problems.